packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -189,6 +237,88 @@ export class DemoComponent implements OnDestroy, OnInit {
       element.click();
       document.body.removeChild(element);
       this.registerDownloadEvent();
+    } else if (this.selectedOutputFormat === "zip") {
+      let zipFile = new JSZip();
+      // Create inner folder
+      const innerFolder = zipFile.folder("readalong");
+      const assetsFolder = innerFolder?.folder("assets");
+      const timestamp = Date.now();
```

Comment:
This gives us a timestamp in seconds since epoch, e.g., `readalong-1710864009240`.
Can we do yyyymmddhhmmss instead? It's just one digit longer but it's human readable.